### ISSUE TITLE
Microwave power tranceiver still not working

### ISSUE THREAD

**Issue by sammhicks** [UmbraSpaceIndustries/MKS#902]
Case 1:
- I've a ship sitting on the runway. It consists of a UKS Power Distribution Module, Some legs, and a level 5 engineer.
- I also have a ship sitting on the launchpad, thus less that 2km away. It has a probe core, i.e. unmanned, and a microwave power tranceiver.
- Power logistics does not occur

Case 2:
- I have the same ship on the runway
- I have a ship sitting on the launchpad. It has a probe core, and a sunflower light.
- Power logistics occurs, even when the sunflower light is not deployed

Case 3:
- I have the same ship on the runway
- I have a ship sitting on the launchpad. It has a MEU-250 drill, a probe core, a kontainer tank, and a microwave power tranceiver.
- Power logistics does not occur.

Case 4:
- I have the same ship on the runway
- I have a ship sitting on the launchpad. It has a UKS pioneer module, a microwave power tranceiver, and a level 5 pilot
- The pioneer module can see the PDU, the microwave power tranceiver cannot

Here are the list of mods the AVC reports:

KSP: 1.1.3 (Win64) - Unity: 5.2.4f1 - OS: Windows 10  (10.0.0) 64bit
USI Tools - 0.7.4
AutoAsparagus - 2.2.1
Chatterer - 0.9.90.1289
Community Resource Pack - 0.5.4
Connected Living Space - 1.2.1.5
DMagic Orbital Science - 1.3.0.1
Contract Parser - 1.0.4
Contracts Window Plus - 1.0.6.4
Progress Parser - 1.0.5
EditorExtensionsRedux - 3.2.14
Firespitter - 7.3
Kerbal Attachment System - 0.5.9
Kerbal Engineer Redux - 1.1.1
Kerbal Inventory System - 1.2.12
kOS - 0.20.1
KSP-AVC Plugin - 1.1.6.1
Docking Port Alignment Indicator - 6.4
StageRecovery - 1.6.4
StationPartsExpansion - 0.4
Cargo Transportation Solutions - 0.4.2
Trajectories - 1.6.2
Kerbal Alarm Clock - 3.7.1
USI Core - 0.2.4
Kolonization Core - 0.2.4
Konstruction - 0.1
USI-LS - 0.4.4
Malemute Rover - 0.1.4
UKS - 0.40.4
[x] Science! - 4.20

**Comment by Tabris83** [comment]
can confirm. the "Power Coupler" built into the UKS parts (MK2, Scout, MK3) work. the "add-on" ones (Microwave Power Transceiver, Power Coupler) do not work

**Comment by jmburbach** [comment]
I found that these parts (transceiver, fuel cell array, etc) do not have a logistics consumer module and so get ignored by underlying logistics code. Adding the logistics consumer module to these parts will get them to work as expected.

**Comment by ranathane** [comment]
OK, just adding:
MODULE
	{
		name = ModuleLogisticsConsumer
	}
does not resolve the issue.

I have a base on the Mun with a PDU and 2 nuke plants. On the PDU is attached a Microwave Power Transmitter (MPT). ~700m away is my first relay station composed of an Anchor Hub, 400es batt, a MK-V Inflatable Storage Module, and another MPT.

![screenshot10](https://cloud.githubusercontent.com/assets/13479555/18214810/a04123b2-711c-11e6-8aeb-580119b1997f.png)

As you can see this appears to be connected. Now ~1950m away from that one I have my next relay station.

![screenshot11](https://cloud.githubusercontent.com/assets/13479555/18214859/d23d5e4e-711c-11e6-8773-ea7d9ddafceb.png)

I even airdropped one of the power packs to add to it thinking maybe it would fix it. Obviously it didn't.
And yes there is a Lvl1 Eng in the PDU.

Please Help!

**Comment by MarkThrimm** [comment]
I have found a workaround to activate the attachable PDUs. Turning them into probe cores by adding:
MODULE
	{
		name = ModuleCommand
		minimumCrew = 0
		RESOURCE
		{
			name = ElectricCharge
			rate = 0.00
		}
	}
made the mat least detect other PDUs in range. I'm fairly new to USI pack, so I'm still testing if the power transmission actually works or not. On a side note, this is the only module that made them detect other PDUs, I've tried multiple combinations. I hope that will help to solve the issue.
